src/Gui/ViewParams.cpp

```diff
@@ -59,3 +65,32 @@ ViewParams *ViewParams::instance() {
     return inst;
 }
 
+void ViewParams::init() {
+    instance()->onDefaultFontSizeChanged();
+}
+
+int ViewParams::appDefaultFontSize() {
+    static int defaultSize;
+    if (!defaultSize) {
+        QFont font;
```

Comment:
Looks like it's not initialized.

Reply:
defaultSize? It's static variable, which is auto init to zero.

Reply:
No, `font` variable ;)

Reply:
@chennes Know that. Probably a matter of taste, but I prefer that it is explicit that we call empty constructor in order to get application default. ;)
Anyway nothing important there.